src/game/line_of_sight/vision.rs

```diff
@@ -130,7 +130,7 @@ pub fn update_visible_squares(
         if ray_start.distance(visible_squares.for_position.coordinates) < 1.0 {
             continue;
         }
-        visible_squares.for_position = *grid_position;
+        //visible_squares.for_position = *grid_position;
```

Comment:
yeah, if we want the rotating in place we could also add an angle to the cache and recompute e.g. every 15 degrees